src/emitter.cpp

```diff
@@ -702,6 +702,9 @@ Emitter& Emitter::Write(const std::string& str) {
       Utils::WriteLiteralString(m_stream, str,
                                 m_pState->CurIndent() + m_pState->GetIndent());
       break;
+    case StringFormat::PreferUnquoted:
+      // error
```

Comment:
If this is an error, should something happen to alert us?

Reply:
That's a fair point. I've replaced the comment with an `assert()` to demonstrate that we shouldn't get here (and alert us if we do).

Reply:
Duh! Your observation points out that PreferUnquoted doesn't need to be in the StringFormat enum at all. Thank you @chri7325